src/tokenizer/index.js

```diff
@@ -555,6 +572,23 @@ export default class Tokenizer extends LocationParser {
     for (let i = 0, e = len == null ? Infinity : len; i < e; ++i) {
       const code = this.input.charCodeAt(this.state.pos);
       let val;
+
+      if (this.hasPlugin("numericSeparator")) {
+        const prev = this.input.charCodeAt(this.state.pos - 1);
+        const next = this.input.charCodeAt(this.state.pos + 1);
+        if (code === 95) {
+          if (forbiddenNumericLiteralSeparatorSiblings.includes(prev) ||
```

Comment:
Thanks for the PR, @rwaldron! I believe we need to switch to `Array#indexOf` here. We don't load any polyfills in Babylon, and the minimum supported node version is 4.0.0.